shared/elements/gaia_grid/style.css

```diff
@@ -211,11 +211,17 @@
   height: 3rem;
 }
 
-.group .header span {
+.group .header span,
```

Comment:
It seems like we shouldn't need this if we've switched to a ```<button>```. Can you remove it if possible so we only have the button selector?